core/src/prediction/LLCausalPredictionStrategy.cpp

```diff
@@ -92,8 +93,8 @@ std::vector<double> LLCausalPredictionStrategy::predict(
     for (size_t j = 0; j < num_variables; ++j){
       size_t current_predictor = linear_correction_variables[j];
       // X - x0 column
-      X(i,j+1) = test_data->get(sampleID, current_predictor)
-                 - original_data->get(index, current_predictor);
+      X(i,j+1) = train_data->get(index, current_predictor) -
```

Comment:
Do we mean to be flipping the sign here? Was this a bug before (or does this not matter)?

Reply:
Doesn't matter as long as you're consistent (LLR can be written both ways)- I flipped it so it looks the same as the code in local linear prediction strategy to avoid confusion, and double checked that the output is the same!

Reply:
Ok, got it.